benchmarks/indexing_example_short.jl

```diff
@@ -0,0 +1,260 @@
+using SpinGlassPEPS
+using MetaGraphs
+using LightGraphs
+using Test
+using TensorCast
+
+@testset "test if the solution of the tensor agreeds with the BF" begin
+
+    #      grid
+    #     A1    |    A2
+    #           |
+    #   1 -- 2 -|- 3
+
+
+    D1 = Dict((1, 2) => 0.652,(2, 3) => 0.73,(3, 3) => 0.592,(2, 2) => 0.704,(1, 1) => 0.868)
+    D2 = Dict((1, 2) => -0.9049,(2, 3) => 0.2838,(3, 3) => -0.7928,(2, 2) => 0.1208,(1, 1) => -0.3342)
+
+    for D in [D1, D2]
+
+        m = 1
+        n = 2
+        t = 2
+
+        L = m * n * t
+        g_ising = ising_graph(D, L)
+
+        # brute force solution
+        bf = brute_force(g_ising; num_states = 1)
+        states = bf.states[1]
+        sol_A1 = states[[1,2]]
+        sol_A2 = states[[3]]
+
+        #particular form of peps tensors
+        update_cells!(
+          g_ising,
+          rule = square_lattice((m, 1, n, 1, t)),
+        )
+
+        fg = factor_graph(
+            g_ising,
+            Dict(1=>4, 2=>2),
+            energy=energy,
+            spectrum = brute_force
+        )
+
+        origin = :NW
+        β = 2.
+
+        x, y = m, n
+        peps = PepsNetwork(x, y, fg, β, origin)
+        pp = PEPSRow(peps, 1)
+
+        # the solution without cutting off
+        M1 = pp[1][1,1,:,1,:]
+        M2 = pp[2][:,1,1,1,:]
+        @reduce MM[a,b] |= sum(x) M1[x,a]*M2[x,b]
+
+        _, inds = findmax(MM)
+
+        h1 = D[(1,1)]
+        h2 = D[(2,2)]
+        J12 = D[(1,2)]
+        J23 = D[(2,3)]
+        h3 = D[(3,3)]
+
+        if D[(1, 2)] == 0.652
```

Comment:
Why should not use this like this. Maybe wrap all this in a funciton, which takes an argument like `verbose`?

Reply:
I have corrected it in my branch, commit 39f7d39, is it ok now?